src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -176,6 +177,32 @@ public function compileIndex(Blueprint $blueprint, Fluent $command)
         );
     }
 
+    /**
+     * Compile a fulltext index key command.
+     *
+     * @param  \Illuminate\Database\Schema\Blueprint  $blueprint
+     * @param  \Illuminate\Support\Fluent  $command
+     * @return string
+     *
+     * @throws \RuntimeException
+     */
+    public function compileFulltext(Blueprint $blueprint, Fluent $command)
+    {
+        if (null === $command->language) {
```

Comment:
```suggestion
        if (is_null($command->language)) {
```

Reply:
A specific reason why you suggest replacing a language construct to a function? The behavior will be the same.

Reply:
Consistency

Reply:
Both ways are used in laravel. I can‘t say which one is the preferred one 🤷‍♂️

Reply:
I see Taylor often converting them to the `is_null` variant.

I don't understand why one does this manually and there's no fixer in place doing this automagically 😅

Reply:
@mfn what fixer do you recommend?

Reply:
@driesvints none actually 😅

sorry for the noise, was thinking about https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/language_construct/is_null.rst but that's the other way around and Style CI can do this too but it's not he CS of the project